src/OpenTelemetry.AutoInstrumentation/Configuration/Settings.cs

```diff
@@ -167,13 +159,13 @@ internal static Settings FromDefaultSources()
 
     private static OtlpExportProtocol? GetExporterOtlpProtocol(IConfigurationSource source)
     {
-        // the defualt in SDK is grpc. http/protobuf should be default for our purposes
+        // the default in SDK is grpc. http/protobuf should be default for our purposes
         var exporterOtlpProtocol = source.GetString(ConfigurationKeys.ExporterOtlpProtocol);
 
         if (string.IsNullOrEmpty(exporterOtlpProtocol) || exporterOtlpProtocol == "http/protobuf")
         {
             // override settings only for http/protobuf
-            // the second part of the condition is needed to override default endpoint as long as https://github.com/open-telemetry/opentelemetry-dotnet/pull/2868 is not fixed
+            // the second part of the condition is needed to override default endpoint as long as we verify if enable System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport in EnvironmentConfigurationHelper
```

Comment:
this could be also addressed via https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/217 if we decide that the user has to set some env var to enable `System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport`

maybe worth leaving a comment?

Reply:
I think that, it strongly depends on the details implementation for #217 and the expected default value. I am adding the comment to the task instead of the code.